api/v1beta1/disruption_webhook.go

```diff
@@ -296,6 +300,24 @@ You first need to remove those chaos pods (and potentially their finalizers) to
 	return nil
 }
 
+func (r *Disruption) validateUserInfo(oldDisruption *Disruption) error {
+	userInfo, err := r.UserInfo()
+	if err != nil {
+		return err
+	}
+
+	oldUserInfo, err := oldDisruption.UserInfo()
```

Comment:
Could we allow overwriting user info if it was previously empty or invalid?

Reply:
Fair point! 👍 Do we allow the creation of a disruption if the user info is empty or invalid? If it is not the case, we can consider the userInfo is valid by default. Hee we avoid the edit or delete of the userInfo,

Reply:
After having a look at the user info webhook, I think it is possible to create a disruption with an empty or an invalid user info 🤔 So I will cover this case to allow a user to override the user info if it is empty or invalid.

Reply:
I am fairly sure it's possible to get an empty one, i've seen it when we fail to set user info due to service accounts not looking how we expect